docs/modules/whats-new/whats-new.adoc

```diff
@@ -0,0 +1,10 @@
+What's new in Platform 5.4
+
+First of all, starting with this release of Hazelcast Platform, the minimum supported Java version has become 17.
```

Comment:
```suggestion
From this release of Hazelcast Platform, the minimum supported Java version is 17.
```

Reply:
Perhaps add something about the benefits of this (better CPU use, additional language features, security updates, and improved support for dev tools (IDEs)